sirepo/job_api.py

```diff
@@ -257,6 +257,10 @@ def _request_content(kwargs):
 
 
 def _run_mode(request_content):
+    def _validate_value(value, *obj_types):
+        assert type(value) in obj_types and value > 0, \
+            f'value={value} expecting a {obj_type} > 0'
```

Comment:
obj_types?

Reply:
Sorry. Was type which I thought might yield funny results with isinstance() because type is a type... so changed it to obj_type (object type). Then realized hours could be a float or an int so changed to obj_types. Now just types:)
https://github.com/radiasoft/sirepo/pull/2511/commits/255438815245fa038582a6c8722f3ec1f8ee52bf